.github/workflows/wasm-pr.yml

```diff
@@ -0,0 +1,40 @@
+name: Skiko Unit tests
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - "jb-main"
+
+concurrency: 
+  group: ${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+  actions: read
+  checks: write
+  pull-requests: write
+
+jobs:
+  unit-tests:
+    runs-on: ubuntu-latest
+    name: Wasm Unit tests on Ubuntu
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'zulu'
+      - name: install Android SDK
+        run: ./jbdeps/android-sdk/downloadAndroidSdk
+      - name: Run Wasm tests
+        run: ./gradlew :compose:foundation:foundation:wasmJsBrowserTest --info
```

Comment:
Jfyi: we have a combined test task - :mpp:testWeb 
It currently tests only runtime on js. We can add more dependencies to it and call here only a combined task.

Not sure if the agents have Chrome installed by default or recent versiont. If Chrome installation is required then you can have a look at Skiko github workflow for web.

Reply:
@eymar pushed relevant changes